[2023-09-13T07:38:45.046+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.054+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.056+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:45.087+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-04 00:00:00+00:00
[2023-09-13T07:38:45.093+0000] {standard_task_runner.py:57} INFO - Started process 143 to run task
[2023-09-13T07:38:45.102+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-04T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpgt80sptu']
[2023-09-13T07:38:45.110+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask extract_data
[2023-09-13T07:38:45.267+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:45.410+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-04T00:00:00+00:00'
[2023-09-13T07:38:45.784+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:45.806+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230904T000000, start_date=20230913T073845, end_date=20230913T073845
[2023-09-13T07:38:45.885+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:45.939+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:19.827+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.845+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-15T11:31:19.852+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:19.883+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-04 00:00:00+00:00
[2023-09-15T11:31:19.889+0000] {standard_task_runner.py:57} INFO - Started process 376 to run task
[2023-09-15T11:31:19.897+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-04T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpfrnssh0a']
[2023-09-15T11:31:19.899+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask extract_data
[2023-09-15T11:31:20.037+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-04T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:20.236+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-04T00:00:00+00:00'
[2023-09-15T11:31:20.571+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:20.600+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230904T000000, start_date=20230915T113119, end_date=20230915T113120
[2023-09-15T11:31:20.648+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:20.679+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
